install/kubernetes/cilium/templates/cilium-operator-podsecuritypolicy.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.operator.psp.enabled }}
```

Comment:
Leftover? we don't have this in the current changes.

Reply:
removed